Description: Add 'Main-Class' to rngconverter JAR file manifest
 The maven-jar-plugin configuration in rngconverter/pom.xml had the intention
 of creating a runnable JAR file, but was not working due to an incorrect
 organization of the XML elements. In particular:
   + the '<manifest>' elt needed to be within the '<archive>' elt
   + the '<addClasspath>' elt needed to be within the '<manifest>' elt.
Author: Alan D. Salewski <salewski@att.net>
Forwarded: no
Last-Update: 2014-03-16
---
--- msv-2009.1+dfsg1.orig/rngconverter/pom.xml
+++ msv-2009.1+dfsg1/rngconverter/pom.xml
@@ -56,11 +56,11 @@ EVEN IF SUN HAS BEEN ADVISED OF THE POSS
 	<configuration>
 	  <archive>
 	    <index>true</index>
+	    <manifest>
+	      <addClasspath>true</addClasspath>
+	      <mainClass>com.sun.msv.writer.relaxng.Driver</mainClass>
+	    </manifest>
 	  </archive>
-	  <manifest>
-	    <mainClass>com.sun.msv.writer.relaxng.Driver</mainClass>
-	  </manifest>
-	  <addClasspath>true</addClasspath>
 	</configuration>
       </plugin>
     </plugins>
